jetson/science_bridge/src/__main__.py

```diff
@@ -61,13 +60,12 @@ def __init__(self):
             "THERMISTOR": self.thermistor_handler,
             "TRIAD": self.triad_handler,
             "TXT": self.txt_handler,
-            "CAROUSEL": self.carousel_handler,
             "HEATER": self.heater_state_handler,
             "AUTOSHUTOFF": self.heater_shutoff_handler
         }
-        self.last_openloop_cmd = -1
         self.max_error_count = 20
         self.sleep = .01
+        self.previous_led_state = LED_state.NONE
```

Comment:
again with the camal case snake case mix...

Reply:
changing to LEDState